README.md

```diff
@@ -74,6 +74,20 @@ steps:
           name: my-repo
 ```
 
+This plugin can be used in combination with the [Docker ECR
+Cache](https://github.com/seek-oss/docker-ecr-cache-buildkite-plugin) plugin to
+reuse a cached base image that is shared with other pipeline steps:
+
+```yaml
+steps:
+  - plugins:
+      - seek-oss/docker-ecr-cache#v1.1.1:
+          target: deps
+      - seek-oss/docker-ecr-publish#v2.0.0:
+          cache-from: ecr://build-cache/org/my-repo
```

Comment:
Hacked-together shorthand for a same-account repository; the underlying URL is retrieved by the plugin. I'm not sure if the `ecr://` prefix is appropriate.

Reply:
how about just `cache`

Reply:
Idea is to use the same name as the [`docker build` option](https://docs.docker.com/engine/reference/commandline/build/#options).

Reply:
The `ecr://` substitution shouldn't break anything given that the underlying Docker command does not expect a protocol:

> [A registry path is similar to a URL, but does not contain a protocol specifier (https://).](https://docs.docker.com/engine/reference/commandline/pull/#pull-from-a-different-registry)